### ISSUE TITLE
Bug: Service config variables not updated when variables are updated

### ISSUE THREAD

**Issue by ociappara** [CybercentreCanada/assemblyline#178]
1. Create a service with below sample config variables

```
config:
  client_id: "abc"
  client_secret: "ThisIsMySecret"

```
2. Add service to Assemblyline
3. Update service config variables like below;

```
config:
  client_id: "def"
  client_secret: "NoLongerASecret"
```

4. Add the new service to Assemblyline, without deleting the previous version
5. Go to the service and pick the new service from the dropdown list
6. New config should be available, however, old variables are still present
7. Same if you add new configs.

**Comment by cccs-sgaron** [comment]
I'm not 100% sure if this is an issue or not and I think I'm going to need just a little more info. For starter I can explain you how the service diffing works so you can tell me if you see things happening differently.

When you register a service to AL, its default config get saved in the DB as `SERVICENAME_vVERSION` so for each service version that where ever installed on your instance, you get the default config for that service. But then we have a service_delta index which is the changes that you are making to the default config. The service delta changes completely override whatever the service default where.

Here's an exemple:

Service_v4.4.0,1 has the following config:
```config:
  client_id: 'abc'
  client_secret: "ThisIsMySecret"
```

Service_v4.4.0.2 has the following config:
```
config:
  client_id: 'def'
  client_secret: "NoLongerASecret"
```

If the service delta is empty (i.e. you did not change the config block from the UI, you will get the proper default for each service. If you changed the client_secret in the UI to `"ThisIsAnUpdatedSecret"`, no matter which of the two service version you pick (4.4.0.1 or 4.4.0.2) the client_secret will always be `"ThisIsAnUpdatedSecret"`. 

Is this what you are observing right now in your system?

**Comment by ociappara** [comment]
No, if I do not change the values in the UI they are still not updated, unless I delete the old service and install the new one.

So I install Service_v4.4.0.1 I don't change anything on the UI, I change the service manifest for Service_v4.4.0.2 and add that to the UI again, the values do not change unless I delete Service_v4.4.0.1 and add the service manifest again to install Service_v4.4.0.2.

This is also true for submission_params, If I add a new submission_params they are not changed on the UI until I delete the service and add it back in again.

**Note**: I only observed this happening when we paste in the service manifest, not when pressing the update button.

**Comment by cccs-sgaron** [comment]
Ok we can check the UI service manifest pasting path.

**Comment by cccs-rs** [comment]
This actually might be related to an issue where the browser is caching the `/whoami` information on app load and so changes that affect the output from that endpoint never make it back to the browser (because it doesn't try to ask for it).

This is something that should be partly resolved in this [PR](https://github.com/CybercentreCanada/assemblyline-ui-frontend/pull/1556). Unfortunately the invalidation that happens is only within the user's context that's making the changes (this doesn't cascade out to all users unless we leverage sockets to communicate that out to all users)
